packages/xml-views-completion/src/providers/elementName/classes.ts

```diff
@@ -156,7 +156,19 @@ function handleDefaultAggregationScenario({
   parentXMLElement: XMLElement;
   parentUI5Class: UI5Class;
 }): classSuggestionContext | null {
-  const defaultAggregation = parentUI5Class.defaultAggregation;
+  let defaultAggregation: UI5Aggregation | undefined = undefined;
+
+  // handle inheritance
+  let currentClass: UI5Class | undefined = parentUI5Class;
+  while (currentClass) {
```

Comment:
@voicis 

Could we wrap this new logic in a function with a clear name and invoke said function?

e.g:
`searchForClosestDefaultAggregation" (better name?).

Basically when I read this function I first want to understand what we are trying to achieve (logical flow)
and only later (optionally) want to see **how** we implemented it.